app/(main)/league/[leagueId]/entry/[entryId]/week/Week.tsx

```diff
@@ -133,6 +138,36 @@ const Week = ({ entry, league, NFLTeams, week }: IWeekProps): JSX.Element => {
     } catch (error) {
       console.error('Could not load week data:', error);
       setError('Could not load week data.');
+    }
+  };
+
+  /**
```

Comment:
Chris, what are your thoughts on adding a spinner to load the previous and current picks? (for a separate ticket of course :))

Reply:
@vmaineng Love the idea! I don't think it is really needed as there is already a spinner on the page and wants the data is loaded (including the previous picks) the page shows.